### DISCUSSION TITLE
How to set web proxy between Java agent and grpc server.

### DISCUSSION THREAD

**Discussion by kinsken** [apache/skywalking#9852] [Q&A] [answered]

**Comment by wu-sheng** [comment, marked as answer]
Why don't you ask for F5 vendor? You are a commercial software customers. This is not our responsibility.

**Reply by kinsken** [reply]
sorry to making you confused.
F5 just provide us a Virtual network service, not care the agent. 
and as I mentioned I tried the connection test using grpcurl, there should be no problem of Proxy side.

could you just tell me if the agent can send data through a proxy? and how to set it?

Thank you

**Reply by wu-sheng** [reply]
We are not providing service at all.
